docs/whats-new/dotnet-docs-2021-07-01.md

```diff
@@ -24,7 +24,7 @@ Welcome to what's new in the .NET docs from July 1, 2021 through July 31, 2021.
 ### New articles
 
 - [Culture creation and case mapping in globalization-invariant mode](../core/compatibility/globalization/6.0/culture-creation-invariant-mode.md)
-- [Implicit `global using` directives in C# projects](../core/compatibility/sdk/6.0/implicit-namespaces.md)
+- Implicit `global using` directives in C# projects (Later removed.)
```

Comment:
I think it would cause less confusion if we just delete this rather than keep it with "later removed," which raises the question "Why?" That said, a July what's new in docs probably doesn't get much traffic, so it doesn't make a big difference.